api-controllers/ServiceFabrikApiController.js

```diff
@@ -639,13 +581,18 @@ class ServiceFabrikApiController extends FabrikBaseController {
       user: req.user
     };
     logger.info('Attempting delete with:', options);
-    return eventmesh.apiServerClient.patchOperationOptions({
+    return eventmesh
+      .apiServerClient.patchOperationOptions({
         operationName: CONST.OPERATION_TYPE.BACKUP,
         operationType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
         operationId: req.params.backup_guid,
         value: options
       })
-      .catchThrow(NotFound, new Gone('Backup does not exist or has already been deleted'))
+      .catch(NotFound, (err) => {
```

Comment:
For restore you dont have to handle restore from object stores? I mean the same logic of getBackup should be applied even for restores right durign the migration period? Why is the file just deleted if no backup info found in etcd?

Reply:
Restore is not yet sfv2

Reply:
ok